server/controllers/profile.js

```diff
@@ -0,0 +1,71 @@
+// We would like to create the following controllers for creating, updating or getting profiles. Include the routes for this ticket as well
+
+// - CREATE
+
+// - Given parameters passed in, create a profile.
+
+// - UPDATE
+
+// - Given an ID and new parameters, update the profile
+
+// - GET
+
+// - Given an ID, return profile with that ID
+
+// - GET all
+
+// - A list of profiles
```

Comment:
I like to see you planning your work beforehand, but after it's completed do we need these comments here still?